### ISSUE TITLE
Which normalization method was used for edgeR (GLM)?

### ISSUE THREAD

**Issue by jdblischak** [bartongroup/profDGE48#2]
I enjoyed reading your thorough [comparison](http://www.ncbi.nlm.nih.gov/pubmed/27022035) of DGE tools and the effect of biological replicates. I was surprised to see the GLM implementation of edgeR produced results that were inconsistent with the other tools (Figure 3, Table 2). As you note in the Discussion, the flexibility of the GLM mode is an important feature.

Looking at the code to implement the various methods, I noticed that the scripts for limma ([line 63](https://github.com/bartongroup/profDGE48/blob/bf60ee1fbef85de25aca80288110af75bcd3688f/DE_tool_scripts/limma.R#L63)) and edgeR (exact) ([line 58](https://github.com/bartongroup/profDGE48/blob/bf60ee1fbef85de25aca80288110af75bcd3688f/DE_tool_scripts/edgeR.R#L58)) use the default TMM normalization, but the script for edgeR (GLM) uses upperquartile normalization ([line 58](https://github.com/bartongroup/profDGE48/blob/bf60ee1fbef85de25aca80288110af75bcd3688f/DE_tool_scripts/edgeRglm.R#L58)). Am I reading the code correctly? If yes, what was the motivation for using a different normalization method for edgeR (GLM)?

**Comment by PietaSchofield** [comment]
Hullo John

Thank you for pointing this out. I was pretty sure when I ran the scripts for the paper I was using the default (TMM) normalisation. I have checked back and regenerated the "gold standard" data for edgeRglm and it appears that the results that went in the paper used TMM for the gold standard. I am waiting for the results of the bootstraps. I suspect that what happened was an edited edgeRglm.R script was submitted to the code repository, as I had been looking at the other normalisations although we did not include those results in the paper.

Pietà
